client/get.go

```diff
@@ -50,18 +50,18 @@ func runGet(cmd *Command, args []string) {
 	if *getTinkKeyset {
 		tinkKeysetInBytes, err := retrieveTinkKeyset(keyID, *getNetwork)
 		if err != nil {
-			fatalf(err.Error())
+			return &ErrorStatus{err, false}
 		}
 		fmt.Printf("%s", string(tinkKeysetInBytes))
-		return
+		return nil
 	}
 	if *getTinkKeysetInfo {
 		tinkKeysetInfo, err := retrieveTinkKeysetInfo(keyID, *getNetwork)
 		if err != nil {
-			fatalf(err.Error())
+			return &ErrorStatus{err, false}
 		}
```

Comment:
I think these two should be true, not sure

Reply:
processed serverError field in corresponding functions and passed the ErrorStatus